### PR TITLE
Minor copy style edits

### PR DESCRIPTION
**Pull request by chasballew**
cc @sandersonet

### CODE REVIEW

File: app/settings/requires-elevation/admin/template.hbs [hunk 1]
```diff
@@ -143,7 +143,7 @@
             </ul>
           {{else}}
             <button {{action 'showOtpRecoveryCodes'}} class="btn btn-primary" type="submit">
-              Show backup codes
+              Show Backup Codes
```
Comment (krallin) [inline app/settings/requires-elevation/admin/template.hbs]: @chasballew you might want to also update the labels here https://github.com/aptible/dashboard.aptible.com/blob/dccac9f41bb4a2c978f4541f7cbeacb7f4642860/app/settings/requires-elevation/admin/controller.js#L48-L61 for this page. let me know if you'd like me to take care of it.

Cheers,

Comment (chasballew) [inline app/settings/requires-elevation/admin/template.hbs]: Thanks - can you help me understand the difference between the first two cases there?

Comment (krallin) [inline app/settings/requires-elevation/admin/template.hbs]: @chasballew sure;

- `Start 2FA activation` is shown when you haven't ever touched 2FA. When you click that, we provide the QR code, prompt you for an OTP token, and show you the next button: `Enable 2FA`
- When you click `Enable 2FA`, we check the OTP token and then enforce 2FA on your account if it's valid.

**Comment by chasballew** [comment]
@krallin - If this is ok, I'll rebase and merge when specs pass.

**Pull request merged**